Detectors/ITSMFT/ITS/macros/test/CheckTopologies.C

```diff
@@ -219,40 +212,40 @@ void CheckTopologies(std::string clusfile = "o2clus_its.root", std::string hitfi
   hComplete->Draw("hist");
   hComplete->Write();
   cComplete->Write();
+
   TCanvas* cNoise = nullptr;
   TCanvas* cSignal = nullptr;
   TH1F* hNoise = nullptr;
   TH1F* hSignal = nullptr;
 
```

Comment:
Since you suppressed the check for labels availability, it crashes if hits file is not provided (i.e. running over "non-MC" data). Could you put the protection back?